Community/Tdarr_Plugin_DOOM_NVENC_Tiered_MKV_CleanAll.js

```diff
@@ -68,6 +112,17 @@ const details = () => {
                 \\nExample 50%:\\n
                 .50`,
       },
+      {
+        name: "bframe",
+        type: 'string',
+        defaultValue: '5',
+        inputUI: {
+          type: 'text',
+        },
+        tooltip: `Set max number of B frames between non-B-frames. Must be an integer between -1 and 16. 0 means that B-frames are disabled. If a value of -1 is used, it will choose an automatic value depending on the encoder. 
+                \\nDefault:\\n
+                5`,
```

Comment:
Max b-frames with an nvidia card is 5. Not 16. `[hevc_nvenc @ 00000224dccb5f00] Max B-frames 16 exceed 5` when running it on my 2080 Super.

Reply:
You're right, I just grabbed these values straight from the ffmpeg docs. Tested it with an Ampere card just now, max b-frames for hevc_nvenc is 5 as well.